x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/vectors/TextEmbeddingQueryVectorBuilder.java

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-package org.elasticsearch.xpack.ml.vectors;
+package org.elasticsearch.xpack.core.ml.vectors;
```

Comment:
This had to move from the `ml` plugin in order to access it in the `inference` plugin. While I could have moved it to `inference`, `core` seemed like a better fit for this standalone class. I did keep the tests in the `ml` plugin however, because they rely on the ML test runner.